examples/balance/input/yes.tsv

```diff
@@ -0,0 +1,1000 @@
+1
```

Comment:
You can replace this with a line in `input/yes.tsv.sh`:
```
seq 1000
```
and `chmod +x input/yes.tsv.sh`.  Same thing applies to `no.tsv`.